android/app/src/main/java/io/locha/p2p/service/ChatService.java

```diff
@@ -46,6 +46,7 @@
     public static final String SERVICE_STOPPED = "com.lochameshchat.SERVICE_STOPPED";
     public static final String CLICK_FOREGROUND_NOTIFICATION = "com.lochameshchat.CLICK_FOREGROUND_NOTIFICATION";
     private String NOTIFICATION_CHANNEL_ID = "com.lochameshchat";
+    private static final String DEFAULT_ADDRESS_LISTEN = "/ip4/0.0.0.0/tcp/4444";
```

Comment:
```suggestion
    private static final String DEFAULT_LISTEN_ADDRESS = "/ip4/0.0.0.0/tcp/4444";
```

Reply:
This could be moved to Runtime.java, make the address parameter nullable and when null use this default address.